demo/values.tpl.yaml

```diff
@@ -1,9 +1,17 @@
 developer:
+  urls:
+    diracx: http://{{ hostname }}:8000
+    minio: http://{{ hostname }}:32000
+    dex: http://{{ hostname }}:32002
+  demoDir: {{ demo_dir }}
   modulesToInstall: {{ modules_to_mount }}
 
 diracx:
   settings:
     DIRACX_SERVICE_AUTH_ALLOWED_REDIRECTS: '["http://{{ hostname }}:8000/docs/oauth2-redirect"]'
+    DIRACX_SANDBOX_STORE_BUCKET_NAME: demo-sandboxes
+    DIRACX_SANDBOX_STORE_S3_CLIENT_KWARGS: '{"endpoint_url": "http://{{ hostname }}:32000", "aws_access_key_id": "console", "aws_secret_access_key": "console123"}'
+    DIRACX_SANDBOX_STORE_AUTO_CREATE_BUCKET: "true"
```

Comment:
I'd keep the analogy with the DB until the end and have an init script create it

Reply:
I implemented that first time but it quickly started causing issues. I think it's much more natural as a settings class: https://github.com/chrisburr/diracx/blob/cbcc2b4bdabba10386f10405ef9d5494b3e6d80b/src/diracx/routers/job_manager/sandboxes.py#L35-L70

We can later debate if we want the automatic bucket creation or something else but I think this is good enough for now and it's easy to change.

Reply:
we can move ahead with that, but here the boundary between what goes into the config and what goes into the env variable starts being a bit blured